Modules/Loadable/Markups/MRML/vtkMRMLMarkupsCurveNode.h

```diff
@@ -83,6 +84,15 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsCurveNode : public vt
   /// \return sum of distances between the curve points, returns 0 in case of an error
   double GetCurveLengthBetweenStartEndPointsWorld(vtkIdType startCurvePointIndex, vtkIdType endCurvePointIndex);
 
+  /// Resample a curve with points constrained to surface
+  /// Projection to surface is constrained by maximumSearchRadius, specified as a percentage of the model diagonal distance
```

Comment:
specify a default value for maximumSearchRadius. Clarify valid range.
"Diagonal distance" -> "size of the model's bounding box diagonal in world coordinate system"